### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Emby.Naming/Video/VideoListResolver.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ public static IEnumerable<VideoInfo> Resolve(IEnumerable<FileSystemMetadata> fil
                 .Resolve(nonExtras).ToList();
 
             var remainingFiles = videoInfos
-                .Where(i => !stackResult.Any(s => i.Path != null && s.ContainsFile(i.Path, i.IsDirectory)))
+                .Where(i => !stackResult.Any(s => s.ContainsFile(i.Path, i.IsDirectory)))
```
Comment (cvium) [inline Emby.Naming/Video/VideoListResolver.cs:41]: Please don't create merge conflicts with my PR

File: Emby.Server.Implementations/Library/CoreResolutionIgnoreRule.cs [hunk 2]
```diff
@@ -52,37 +52,31 @@ public bool ShouldIgnore(FileSystemMetadata fileInfo, BaseItem parent)
...
-                        return true;
-                    }
+                if (string.Equals(filename, BaseItem.ThemeSongsFolderName, StringComparison.OrdinalIgnoreCase))
+                {
+                    return true;
                 }
             }
             else
             {
-                if (parent != null)
```
Comment (cvium) [inline Emby.Server.Implementations/Library/CoreResolutionIgnoreRule.cs:78]: Parent can be null

File: Emby.Server.Implementations/Library/LibraryManager.cs [hunk 3]
```diff
@@ -677,7 +676,7 @@ public IEnumerable<BaseItem> ResolvePaths(IEnumerable<FileSystemMetadata> files,
                 {
                     var result = resolver.ResolveMultiple(parent, fileList, collectionType, directoryService);
 
-                    if (result != null && result.Items.Count > 0)
+                    if (result.Items.Count > 0)
```
Comment (cvium) [inline Emby.Server.Implementations/Library/LibraryManager.cs:679]: Result can be null

File: Emby.Dlna/Server/DescriptionXmlBuilder.cs [hunk 4]
```diff
@@ -189,7 +189,7 @@ private void AppendIconList(StringBuilder builder)
                 builder.Append("<icon>");
 
                 builder.Append("<mimetype>")
-                    .Append(SecurityElement.Escape(icon.MimeType ?? string.Empty))
+                    .Append(SecurityElement.Escape(icon.MimeType))
```
Comment (cvium) [inline Emby.Dlna/Server/DescriptionXmlBuilder.cs:192]: This may be null
image

**Review by cvium — APPROVED** [review]

**Review by crobibero — APPROVED** [review]

**Comment by crobibero** [comment]
@jellyfin-bot rebase

**Pull request merged**